README.md

```diff
@@ -1,2 +1,4 @@
 # code-review-test-repo
```

Comment:
A __second__ single test comment on an edit (added via "Add single comment").

Reply:
Forgot to add comment before resolution. Adding now.